arviz/data/inference_data.py

```diff
@@ -195,8 +197,10 @@ def _repr_html_(self) -> str:
                         for group in self._groups_all
                     ]
                 )
-                formatted_html_template = (  # pylint: disable=possibly-unused-variable
-                    HtmlTemplate.html_template.format(elements)
+                formatted_html_template = (
+                    HtmlTemplate.html_template.format(  # pylint: disable=possibly-unused-variable
+                        elements
+                    )
```

Comment:
is this due to latest black or some older version?

```suggestion
                formatted_html_template = (    # pylint: disable=possibly-unused-variable
                    HtmlTemplate.html_template.format(elements)
```

you can move the pylint to correct line manually

```suggestion
                formatted_html_template = (    # pylint: disable=possibly-unused-variable
                    HtmlTemplate.html_template.format(
                        elements
                    )
```

Reply:
For our repos I fixed my version to black@19.10b0 because we had some problems with black version mismatches and disliked the newer line length. It defaulted to use 20.8b1 now after I force installed all requirements.